WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -133,7 +109,7 @@ private bool CheckForFlags(ApiResponseItem response)
 		bool shouldBan = false;
 
 		var coinflaglist = response.Cscore_section.Cscore_info;
-		shouldBan = coinflaglist.Any(cscoreInfo => _flagIds.Contains(cscoreInfo.Id));
+		shouldBan = coinflaglist.Any(cscoreInfo => WabiSabiConfig.RiskFlags?.Contains(cscoreInfo.Id) is true);
```

Comment:
The flag should be contained by the list - otherwise the web and the config is not consistent. I would check if we got any flag - if no then the coin is clean. If we got back a flag but it is not in the list then the coin is clean but logwarning that the flag coming from the remote site is unknown for the backend.

Reply:
If we have no flags defined in the config, the verifier won't be created.
https://github.com/zkSNACKs/WalletWasabi/blob/fed272bdacecba85003e777da5e8d51cddbe8738/WalletWasabi.Backend/Global.cs#L101-L104

Reply:
Logging out the unknown ID is a good idea, I will do it quickly.